_site/authors/etagwerker/index.html

```diff
@@ -4,41 +4,48 @@
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
 
-    
 
-    <title>OmbuLabs Blog - The Lean Software Boutique - Author: Etagwerker</title>
+{"layout"=>"default", "author"=>"etagwerker", "title"=>"Articles by Ernesto Tagwerker", "content"=>"{% assign author = site.authors[page.author] %}\n<h2 class=\"author-name\">Articles by <span class=\"tag\">{{ author.display_name }}</span>\n</h2>\n<h3 class=\"author-description\"><a href=\"http://twitter.com/{{author.twitter}}\"\n                                      title=\"{{author.display_name}} on Twitter\"\n                                      target=\"_blank\">Follow @{{ page.author }} on Twitter</a></h3>\n\n<main id=\"content\" class=\"content category\">\n{% for post in site.posts %}\n  {% if post.author == page.author %}\n    {% include post_excerpt.html %}\n  {% endif %}\n{% endfor %}\n</main>\n", "dir"=>"/authors/etagwerker", "name"=>"index.html", "path"=>"authors/etagwerker/index.html", "url"=>"/authors/etagwerker/index.html"}
```

Comment:
@etagwerker this is also repeated throughout all of the generated pages, including posts.